app/frontend/pages/Contribution/Form.tsx

```
@@ -0,0 +1,224 @@
+      return false
+    }
+    return true
+  }
+
+  function validateContribution (): boolean {
+    const dataForValidation = {
+      contribution: {
+        ...data.contribution,
+        // files: Array.isArray(data.contribution.files) ? data.contribution.files : []
+        files: data.contribution.files instanceof Array
+          ? data.contribution.files.filter((file): file is File => file instanceof File)
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)